pkg/minikube/cluster/cluster.go

```diff
@@ -80,7 +80,10 @@ func StartHost(api libmachine.API, config cfg.MachineConfig) (*host.Host, error)
 	}
 
 	if h.Driver.DriverName() != config.VMDriver {
-		fmt.Printf("Skipping %s driver, existing host has %s driver.\n", config.VMDriver, h.Driver.DriverName())
+		console.Warning("Ignoring --vm-driver=%s, as the %q host was created using the %s driver.",
+			config.VMDriver, cfg.GetMachineName(), h.Driver.DriverName())
+		console.Warning("To change drivers, create a new host using `minikube start -p <name>` or run `minikube delete -p %s`",
```

Comment:
missing a `%s`

error message: 

``
⚠️  Ignoring --vm-driver=virtualbox, as the "minikube" host was created using the hyperkit driver.
⚠️  To change drivers, create a new host using `minikube start -p <name>` or run `minikube delete -p hyperkit`%!!(MISSING)!(MISSING)(EXTRA string=minikube)
``

Reply:
Good find. New text reads:

```
⚠️  Ignoring --vm-driver=virtualbox, as the existing "minikube" VM was created using the hyperkit driver.
⚠️  To switch drivers, you may create a new VM using `minikube start -p <name> --vm-driver=virtualbox`
⚠️  Alternatively, you may delete the existing VM using `minikube delete -p minikube`
```